packages/core/src/shared/errors.ts

```diff
@@ -391,7 +390,9 @@ export function scrubNames(s: string, username?: string) {
             }
         }
 
-        r += ` ${start.replace(/\\/g, '/')}${scrubbed.replace(/^[\/\\]+/, '')}${fileExt}`
+        // includes leading '.', eg: '.json'
+        const fileExt = pathSegments[pathSegments.length - 1].match(fileExtRe) ?? ''
+        r += ` ${start.replace(/\\/g, '/')}${scrubbed.replace(/^[\/\\]+/, '')}${fileExt.length < 6 ? fileExt : ''}`
```

Comment:
since `fileExtRe` is defined with `{1,4}` , do we still need the `fileExt.length < 6` check?